### DISCUSSION TITLE
Trying to add a sparsity loss to learned coefficients for inverse problems

### DISCUSSION THREAD

**Discussion by sheikhu28** [lululxvi/deepxde#1240] [Q&A]
As the title says, I am trying to add a sparsity loss to the learned parameters corresponding to the inverse problem in question. Any ideas on how to do this efficiently in DeepXDE?

**Comment by lululxvi** [comment]
What is sparsity loss?

**Reply by sheikhu28** [reply]
My bad, I meant something like an L1 penalty on the learned coefficients in hopes of encouraging sparsity among them.

**Reply by lululxvi** [reply]
Then use L1 regularization.
